extensions/bigtable/install_hbase_bigtable.sh

```diff
@@ -78,3 +80,48 @@ echo -e "YARN_OPTS=\"\${YARN_OPTS} -Dyarn.app.mapreduce.am.command-opts=\"${BIGT
 # the hbase classpath.
 echo -e "HBASE_OPTS=\"\${HBASE_OPTS} ${BIGTABLE_BOOT_OPTS}\"" >> "${HBASE_CONF_DIR}/hbase-env.sh"
 
+# Configure Spark
+if [ -d "/home/hadoop/spark-install/" ]; then
+    SPARK_HOME="/home/hadoop/spark-install"
+    export MASTER=spark:\/\/${MASTER_HOSTNAME}:7077
+    export SPARK_LOCAL_IP=${MASTER_HOSTNAME}
+    export SPARK_MASTER_HOST=${MASTER_HOSTNAME}
+    export SPARK_CONF_DIR=${SPARK_HOME}\/conf
+    export SPARK_MASTER_OPTS=" -Xbootclasspath/p:${ALPN_CLASSPATH}"
+    export SPARK_DAEMON_JAVA_OPTS=" -Xbootclasspath/p:${ALPN_CLASSPATH}"
+    export SPARK_EXECUTOR_OPTS=" -Xbootclasspath/p:${ALPN_CLASSPATH}"
+    export SPARK_WORKER_OPTS=" -Xbootclasspath/p:${ALPN_CLASSPATH}"
+
+    PATH=${SPARK_HOME}/bin:$PATH
+    export BIGTABLE_CLASSPATH
+    # Merge spark-install/conf/core-site.xml with its template
+    bdconfig merge_configurations \
```

Comment:
Is there a way to have separate configuration XML files for each of these? I don't know if I like putting everything into hadoop-install/.../conf/core-site.xml. 

I know this was added as a result of another comment, but I was thinking more along the lines of 
spark/conf/core-site.xml, spark/conf/hbase-site.xml, spark/conf/spark-site.xml and each file having only what is needed for that component.

I think seeing a bunch of extra configuration in core-site will be confusing and could potentially lead to issues if values from one confiugration file overwrite those in another.